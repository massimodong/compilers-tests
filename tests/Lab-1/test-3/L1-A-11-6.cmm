struct pQLjnmm ;
float TACPXbVPPF55qTMa ( ) {
}
int a0xSwZ3 [ 889668607 ] 
;
int k ( struct K9rC x8GVhyYcSK ) {
  struct {
  }uiMqrmaFGEr7IkP64nWW ;
  if ( KeLLVoK9U1j3lU )   JEqX = u5mfmpRJe || jY0 
    > qilwjWOrEOKulbARdPQ + JCTd6xYrtZk * IF ( XoytY && 
    F4tF >= ! - NkrskD9 [ OaUX <= Wo8AsexxC + whtTLToQb0i 
    / SqB3OaTNNpS3 ( smgynEyZsHqZN , HlhOQ8m20VckJw4a82 
    , b4cRe1U65 = aW1aMOm1d2dyPaZdeupA || c5KfRVJHAbwIKToZ 
    && wB5OCoE < * ! ! a9dpt5WXUAIl6qsnRZaF ( m25 != ( 
    ecD222 = fSokEE2TlIeY == vTnLcerkp - ( hn4kPUMcZtrfXCC2 
    ) + - kt3iFKFb - QYRDhvzL7HqrgfR ) ) + 1108589541 
    . hu_aAEXGQgS ) ] - 681553166.1481878505 ) / ! - pIiCm9XCw 
    ( ) . zn ;

}
int ;
struct oS9H8CXIuu7UnuDWSma ;
struct o1rdXg {
};
else int ;
int ;
int ;
int b7pjRYwWxhf1XzNS ( int POxzf8GiPBLC , int G3D 
  ) {
  while ( wp7TBC045Wa_SAaD ) {
    int _RLlnm , jKTUUkItH = a6R3 ;
  }
}
int ;
int KHVTl ( ) {
  Bq2CnK8Q_07W ;
  L9 ;
  E1fboD ;
  if ( nwQzPLKnRbzBOJGT )   cnCcfsDHJoJcUHL ;
else   if ( ew )   if ( G )   EzjlLU3LYHzi ;
else   S102_NniM_AXKcUA27 ;

else   if ( shkcxTsB )   if / ( MB8qcEfPDF1yob )   return 
    LtRR5F ;




}
int ;
int ;== 
int Sn , oWvwoSKTgyTdes3 

;
int ;
int ;
