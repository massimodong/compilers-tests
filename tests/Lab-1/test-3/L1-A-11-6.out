Error Type B at Line 14
Error Type B at Line 25
Error Type B at Line 43
Error Type B at Line 51
